src/frontend/efiling-frontend/src/components/page/cso-account/CSOAccount.js

```diff
@@ -50,11 +51,7 @@ export default function CSOAccount({
         setCsoAccountStatus({ exists: true, isNew: true });
       })
       .catch(error => {
-        const errorUrl = sessionStorage.getItem("errorUrl");
-        window.open(
-          `${errorUrl}?status=${error.response.status}&message=${error.response.data.message}`,
-          "_self"
-        );
+        errorRedirect(sessionStorage.getItem("errorUrl"), error);
```

Comment:
👚 clean

Reply:
i dont even think we need to pass in this first param here, the function can be responsible for pulling it from sessionStorage